lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -228,6 +247,50 @@ public function execute($params = null)
         return true;
     }
 
+    /**
+     * Handle $this->_longData after regular query parameters have been bound
+     *
+     * @throws MysqliException
+     */
+    private function processLongData()
+    {
+        foreach ($this->longData as $paramNr => $value) {
+            if (! is_resource($value)) {
```

Comment:
Unfinished discussion: https://github.com/doctrine/dbal/pull/3217#discussion_r217223893.

Reply:
See above.

Reply:
Please link to the comment.

Reply:
https://github.com/doctrine/dbal/pull/3217#discussion_r217895850 (I wanted to keep the discussion together in one thread.)